### DISCUSSION TITLE
Swap icons with flip effect: horizontal instead of vertical effect

### DISCUSSION THREAD

**Discussion by Seranged** [saadeghi/daisyui#647] [Ideas / feature requests]
Hi

Any chance we can have the alternative effect added?

Currently the effect flips vertically left to right.

The opposite (horizontally, top to down) would be awesome

Thank you !

<img width="917" alt="image" src="https://user-images.githubusercontent.com/80223622/159997736-a4eafaec-17f8-4ab6-88f9-bac5128e2693.png">

**Comment by saadeghi** [comment]
May I ask what is the use case for it?

**Reply by Seranged** [reply]
Of course !

I'm going to alternate the rendering of two field inputs and wanted a visual on the input flipping, the fields flipping horizontally rather than vertically would make for a better UX

If it's a lot of hassle i can keep it vertically. It's not a huge deal.

Thanks Saadeghi !

**Comment by saadeghi** [comment]
The problem is, there are [no limits](https://animate.style/) on how many animation effects can be included for this and I don't want to increase the library file size with animation effects. But you can always extend current style and customize it yourself.

Here's what you're looking for: https://play.tailwindcss.com/z7ncVn0Ily?file=css

**Reply by Seranged** [reply]
Ty sir !
